crypto/dilithium/pqcrystals_dilithium_ref_common/poly.h

```diff
@@ -1,26 +1,26 @@
-#ifndef POLY_H
-#define POLY_H
+#ifndef ML_DSA_POLY_H
+#define ML_DSA_POLY_H
 
 #include <stdint.h>
 #include "params.h"
 
 typedef struct {
-  int32_t coeffs[N];
+  int32_t coeffs[ML_DSA_N];
 } poly;
 
-void poly_reduce(poly *a);
+void ml_dsa_poly_reduce(poly *a);
 
 void poly_caddq(poly *a);
```

Comment:
Could prefix these for consistency, but also not necessary and deeply internal.

Reply:
I prefer prefix everything for consistency and avoiding any future confusion.

Reply:
done [2bbdfc9](https://github.com/aws/aws-lc/pull/2072/commits/2bbdfc975539e79985135f515caa45399229d1f6) for everything